modules/repository-datastax/db/mutation/V035__Create_ExperimentTag_table.cql

```diff
@@ -0,0 +1,5 @@
+create table experiment_tag (
```

Comment:
New location for cassandra delta is "modules/repository-datastax/src/main/resources/com/intuit/wasabi/repository/impl/cassandra/migration"